src/TestRunner.js

```diff
@@ -69,6 +69,7 @@ class TestRunner extends EventEmitter {
     this._app = null
     this._server = null
     this._browser = null
+    this._puppeteerOptions = { options }
```

Comment:
This should be this:
```
this._puppeteerOptions = options.puppeteerOptions
```

I would pull this below line 74 though, so that you can take advantage of destructuring.